PipelinesTasks/PRMetrics/buildAndReleaseTask/Updaters/iMetrics.ts

```diff
@@ -0,0 +1,7 @@
+export interface IMetrics {
```

Comment:
I like this! Nice conversion away from the Hashtable into a more OOP design. Thanks!